apps/web/src/pages/activities/services/chart-bar/data.service.ts

```diff
@@ -17,15 +17,18 @@ export function getChartData(data: IActivityGraphStats[] | undefined, isDark: bo
   return buildChartDataContainer(data, isDark);
 }
 
-function buildChartDataContainer(data: IActivityGraphStats[], isDark: boolean): IChartData {
+function buildChartDataContainer(data: IActivityGraphStats[], isDark: boolean): any {
   return {
-    labels: buildChartDateLabels(data),
     datasets: [
       {
         backgroundColor: isDark ? colors.B20 : colors.BGLight,
         hoverBackgroundColor: createGradientColor(),
         data: buildChartData(data),
         borderRadius: 7,
+        parsing: {
+          xAxisKey: 'dateLabel',
+          yAxisKey: 'count',
+        },
```

Comment:
changed this to be like this because before the "data item" was the count, now it's the object (shown in the above comment)